api/core/core.go

```diff
@@ -19,6 +19,7 @@ import (
 	"fmt"
 
 	"github.com/open-telemetry/opentelemetry-go/api/unit"
+	"strconv"
```

Comment:
nit: this should be in the above section

Reply:
done.